includes/polling/ports/procera.inc.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+* procera.inc.php
+*
+* LibreNMS Procera Ports include
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*
+* @package    LibreNMS
+* @link       http://librenms.org
+* @copyright  2017 Paul Heinrichs
+* @author     Paul Heinrichs <pdheinrichs@gmail.com>
+*/
+
+$packetlogic_stats = snmpwalk_cache_oid($device, 'netDeviceTable', array(), 'PACKETLOGIC-CHANNEL-MIB');
+$packetlogic_info = snmpwalk_cache_oid($device, 'channelInfoTable', array(), 'PACKETLOGIC-CHANNEL-MIB');
```

Comment:
If you put both these in the same array it will make things easier for you below.  Just send the array from the first walk to the second walk instead of array().

snmpwalk_group() is a newer function, but either is fine here.

Reply:
I think I'd like to see the use of `snmpwalk_group()` if possible.

Reply:
I attempted the snmpwalk_group(), I don't know if the oddly decimal labeled indexes are causing issues but here's what happened, also created some odd loop that created 6 more interfaces :confused: 

https://pastebin.com/eADLHYij

Did I use it wrong?

Reply:
That is odd @pheinrichs can you do that same thing with debug set so I can see the raw snmpwalk output?

Reply:
Adding the depth at 2 seems to be kind of working but still giving the array a sub array with an index of [.0] 

https://pastebin.com/0gSBzhwq

Reply:
Yeah, the snmp engine isn't returning it right.

It is returning `channelInternalTxAborted[11].0` when it should be `channelInternalTxAborted[11][0]`

Maybe I can update the code to handle this as it might be common when walking tables of single values.

Reply:
#7427 should fix this